connect/runtime/src/main/java/org/apache/kafka/connect/runtime/AbstractHerder.java

```diff
@@ -189,6 +190,16 @@ public ConfigInfos validateConfigs(String connType, Map<String, String> connecto
         return generateResult(connType, resultConfigKeys, configValues, allGroups);
     }
 
+    @Override
+    public List<String> connectorPlugins() {
+        List<String> connectorPlugins = new LinkedList<>();
+        Set<Class<? extends Connector>> connectorClasses = Worker.getConnectorPlugins();
```

Comment:
You can just do something like `return new ArrayList<String>(Worker.getConnectorPlugins())`. No need to manually copy with a for loop.